jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/gzip/GzipHandler.java

```diff
@@ -85,13 +85,27 @@ else if (type.startsWith("image/") ||
                 type.startsWith("video/"))
                 _mimeTypes.exclude(type);
         }
+
         _mimeTypes.exclude("application/compress");
+        _paths.exclude("*.z");
```

Comment:
That seems a very broad brush stroke. Also I don't see this mimetype listed on https://www.iana.org/assignments/media-types/media-types.txt

Reply:
This was taken from our mime.properties file.

Reply:
yikes